exercises/practice/linked-list/linked-list.spec.wren

```diff
@@ -0,0 +1,213 @@
+import "wren-testie/testie" for Testie, Expect
+import "./linked-list" for LinkedList
+
+Testie.test("Linked List") { |do, skip|
+
+  do.test("pop gets last element from the list") {
+    var list = LinkedList.new()
+    list.push(7)
+    Expect.value(list.pop()).toEqual(7)
+  }
+
+  do.test("push/pop add and remove at the end of the list respectively") {
```

Comment:
Shouldn't all of these be `skip.test` other than the first?

Reply:
Done